TricksterBots/Bots/Bridge/BridgeBot.cs

```diff
@@ -67,14 +67,16 @@ public BidBase SuggestBid(BridgeBidHistory history, Hand hand)
                 //  when opening, prefer suggestions with higher minimum points first, then higher minimum cards
             {
                 suggestions = suggestions
-                    .OrderByDescending(s => s.why.Points.Min)
+                    .OrderByDescending(s => s.why.Priority)
+                    .ThenByDescending(s => s.why.Points.Min)
                     .ThenByDescending(s => s.why.HandShape.Max(hs => hs.Value.Min));
             }
             else
                 //  in other phases, prefer finding the best fit first (prioritizing majors), then higher minimum points
             {
                 suggestions = suggestions
-                    .OrderBy(s => s.why.HandShape.Any(hs => IsMajor(hs.Key) && hs.Value.Min > 3) ? 0 : 1)
+                    .OrderByDescending(s => s.why.Priority)
+                    .ThenBy(s => s.why.HandShape.Any(hs => IsMajor(hs.Key) && hs.Value.Min > 3) ? 0 : 1)
```

Comment:
These changes introduce the `Priority` field as the top consideration when choosing between multiple, valid bids regardless of phase. When `Priority` is equal (still the case for everything except Stayman today), we'll fall back to the existing ordering.